app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Here will go the data about the specific user</h1>
+<html>
+  <head>
+    <%= stylesheet_link_tag 'user' %>
+
+  </head>
+
+  <body>
+
+  <%= render 'shared/user_card', user: @user %>
+
+<div class="userBio">
+    <h4> Bio</h4>
+    <p><%= @user.bio %></p>
+</div>
+
+<%= render partial: 'shared/posts_container', locals: { posts: @user.posts } %>
+
+
+<div>
+  <div class="seeAllPostBtn">
+    <button >See all posts</button>
```

Comment:
- [ ] Please make sure that clicking on the **See all Post** link opens the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) :pray:  you can implement this like :point_down: 
  ```ruby
  <button>
  <%= link_to 'See all Post', user_posts_path(@user) %>
  </button>
  ```

Reply:
@Owusu-Desmond  done!

thanks for the suggestion!